services/submission/src/main/java/app/coronawarn/server/services/submission/verification/VerificationServerClient.java

```diff
@@ -0,0 +1,10 @@
+package app.coronawarn.server.services.submission.verification;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@FeignClient(name = "verification-server", url = "${services.submission.verification.base-url}")
+public interface VerificationServerClient {
+  @PostMapping(value = "${services.submission.verification.path}", consumes = "application/json")
+  String verfiyTan(Tan tan);
```

Comment:
typo - also would be nice to use `MediaType.APPLICATION_JSON` here instead of the string literal